pkg/relay.go

```diff
@@ -306,8 +314,8 @@ func (rs *DefaultRelay) GetPayload(ctx context.Context, payloadRequest *types.Si
 		payloadRequest.Signature[:],
 	)
 	if !ok || err != nil {
-		rs.Log().WithField(
-			"pubKey", proposerPubkey,
+		logger.WithField(
+			"proposer", proposerPubkey,
```

Comment:
can we make this pub_key and the msg say "proposer signature invalid. We should actually value that convention across the rest of the repo